### ISSUE TITLE
Dashed lines before labeled hour-segments are not visible

### ISSUE THREAD

**Issue by l0wlik34G6** [mattlewis92/angular-calendar#952]
**Describe the bug**

Dashed line is not visible before labeled hour-segments (see screenshot)

I'm pretty sure, this does not fit every way of displaying the calendar, but for me it was ok:
```
   .cal-day-view .cal-hour:not(:last-child) .cal-hour-segment,
   .cal-day-view .cal-hour:last-child :not(:last-child) .cal-hour-segment {
+    z-index: 150;
+    position: relative;
     border-bottom: thin dashed; }
   .cal-day-view .cal-time {
```
```
   .cal-day-view .cal-event-container {
     position: absolute;
+    z-index: 200;
     cursor: pointer; }
     .cal-day-view .cal-event-container.resize-active {
-      z-index: 1;
+      z-index: 190;
       pointer-events: none; }
   .cal-day-view .cal-event {
```
```
     border-bottom-right-radius: 5px; }
   .cal-day-view .cal-drag-active {
-    z-index: 1; }
+    z-index: 190; }
     .cal-day-view .cal-drag-active * {
       pointer-events: none; }
```

Hopefully, I didn't forget any changes I made.

**Screenshots**

Before:
![image](https://user-images.githubusercontent.com/48749340/56161545-11500480-5fca-11e9-87c4-45a42b1f53b4.png)
(to see the issue better, I changed the border color to black)

After:
![image](https://user-images.githubusercontent.com/48749340/56161950-24af9f80-5fcb-11e9-992f-9841d14161ab.png)

**Versions**

- `@angular/core`: 7.2.13
- `angular-calendar`: 0.27.3
- Browser name and version: chrome 73.0

**Comment by mattlewis92** [comment]
I'm also unable to reproduce this as well, can you make a runnable stackblitz for this as well please? Thanks!

**Comment by l0wlik34G6** [comment]
Also here: https://stackblitz.com/edit/angular-calendar-952
:)

**Comment by mattlewis92** [comment]
Got it, thanks! Again it looks like another bootstrap issue

**Comment by mattlewis92** [comment]
Also fixed in `0.27.4` 👍
